packages/ui/src/utils/to-escaped-id.ts

```diff
@@ -3,8 +3,10 @@
  */
 export function toEscapedId(id: string): string {
   return id
-    .replace("(", "_")
-    .replace(")", "_")
-    .replace(",", "_")
-    .replace(" ", "_");
+    .replaceAll("(", "__)")
```

Comment:
Is it safe for me to remove the `)` in `"__)"`, or is there a move I am not seeing?

I think line 7 replaces it.

Reply:
OMG! That explains why i struggled so much with those tests! It's just a typo